.github/workflows/test.yml

```
@@ -0,0 +1,16 @@
+name: Sinbad Test Workflow
+on:
+  pull_request:
+    branches:
+    - master
+jobs:
+  build_and_test:
+    name: Build and Test
+    runs-on: ubuntu_latest
```

@yaswanth-deriv this has to be `ubuntu-latest` 
please fix it.
```suggestion
    runs-on: ubuntu-latest
```